kpops/components/base_components/base_defaults_component.py

```diff
@@ -165,20 +170,23 @@ def defaults_from_yaml(path: Path, key: str) -> dict:
     """
     content = load_yaml_file(path, substitution=ENV)
     if not isinstance(content, dict):
-        raise TypeError(
+        msg = (
             "Default files should be structured as map ([app type] -> [default config]"
         )
+        raise TypeError(
+            msg,
```

Comment:
tbh not a fan of the forced trailing comma :( this could easily be a one liner

Reply:
This probably happened because it used to be indented like that before. If I make it a one-liner it will not be flagged

Reply:
Tested it now, `black` did it, because of the forced comma

Reply:
This will be flagged and formatted by both `black` and `ruff format`
```
    def apply_component(
        self, component_class: type[PipelineComponent], component_data: dict,
    ) -> None:
```

This won't:
```
    def apply_component(
        self, component_class: type[PipelineComponent], component_data: dict
    ) -> None:
```

Reply:
do we have to force the trailing comma though?

Reply:
How can I revert just the [COM](https://github.com/bakdata/kpops/pull/363/commits/301284eb02dbb707fefa5ab50476b06ebee757ff) commit?

Reply:
`git revert 301284eb02dbb707fefa5ab50476b06ebee757ff`?

Reply:
WDYT about getting rid of [COM812](https://docs.astral.sh/ruff/rules/missing-trailing-comma/), but keeping [COM818](https://docs.astral.sh/ruff/rules/trailing-comma-on-bare-tuple/) and [COM819](https://docs.astral.sh/ruff/rules/prohibited-trailing-comma/)

Reply:
good idea, that looks ideal